### PR TITLE
alarm config support i18n

### LINKED ISSUE #7076: oap alarm config can support i18n？
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [ ] Question or discussion
- [ ] Bug
- [ ✓ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?

___
### Bug
- Which version of SkyWalking, OS, and JRE?

- Which company or project?

- What happened?
If possible, provide a way to reproduce the error. e.g. demo application, component version.

___
### Requirement or improvement
- Please describe your requirements or improvement suggestions.

### PR DESCRIPTION
**Pull request by lmx1989219**
- [✔ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #7076.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java [hunk 1]
```diff
@@ -264,16 +268,16 @@ private void readFeishuConfig(Rules rules) {
     @SuppressWarnings("unchecked")
     private void readWeLinkConfig(Rules rules) {
         Map<String, Object> welinkConfig = (Map<String, Object>) yamlData.getOrDefault(
-            "welinkHooks",
-            Collections.EMPTY_MAP
+                "welinkHooks",
```
Comment (wu-sheng) [inline oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java:271]: Unexpected change, please revert and polish changes.

File: oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/Rules.java [hunk 2]
```diff
@@ -45,6 +47,7 @@
     private WeLinkSettings welinks;
 
     public Rules() {
+        this.language = "cn";
```
Comment (wu-sheng) [inline oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/Rules.java:50]: Why cn?

**Review by wu-sheng — COMMENTED** [review]
What is the point of doing global i18n? This seems as same as supporting Chinese text in alarm message only.

File: oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java (hunk 1 above)
Comment (lmx1989219) [reply oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java:271]: @wu-sheng yes, now its only a simply translate.

Comment (lmx1989219) [reply oap-server/server-alarm-plugin/src/main/java/org/apache/skywalking/oap/server/core/alarm/provider/RulesReader.java:271]: i would be conside i18n implements deeply ,now i close this pr.

**Pull request closed without merging**